File: banda/JCZhang-Banda/JCZhang-Banda-JAVA/src/main/java/models/BassGuitarist.java
package models;

import interfaces.IBassGuitarist;

public class BassGuitarist extends Musician implements IBassGuitarist {

    String name;
    int yearsOfExperience;
    int numberOfStrings;
    static double salary = (Musician.salary * 1.5);


    public BassGuitarist(String name, int yearsOfExperience, int numberOfStrings) {
        super(name, yearsOfExperience);
        this.name = name;
        this.yearsOfExperience = yearsOfExperience;
        this.numberOfStrings = numberOfStrings;

    }

    @Override
    public void playBass() {
        System.out.println("Im a bass guitarist and im playing bass guitar");
    }
}

File: banda/JCZhang-Banda/JCZhang-Banda_KOTLIN/src/main/kotlin/models/BassGuitarist.kt
package models

import interfaces.IBassGuitarist

class BassGuitarist(override var name: String?, override var yearsOfExperience: Int, var numberOfStrings: Int) :
    Musician(name, yearsOfExperience), IBassGuitarist {
    override fun playBass() {
        println("Im a bass guitarist and im playing bass guitar")
    }

    companion object {
        var salary: Double = Musician.salary * 1.5
    }
}